### DISCUSSION TITLE
Recommended comments library for using with Next.js

### DISCUSSION THREAD

**Discussion by PritamSangani** [VirtusLab-Open-Source/strapi-plugin-comments#74] [Q&A]
Hi! I am wondering whether there is a recommended comments library to use with Next.js to integrate with this comments plugin as I don't really have capacity to build my own comments ui?

**Comment by cyp3rius** [comment]
With one of client project we've been using Strapi + Next.js + this plugin to manage comments. Depends what requirements you've got?
